PowerShell/Module/Private/_DeploymentFunctions.ps1

```diff
@@ -194,7 +194,19 @@ function _deployProject
 
         $amazonLambdaToolsPath = _configureAmazonLambdaTools
 
-        $env:AWS_EXECUTION_ENV="AWSLambdaPSCore"
+        $moduleVersion = "Unknown"
+        $module = $MyInvocation.MyCommand.Module
+        if ($module) {
+            $moduleVersion = $module.Version.ToString()
+        }
+        
+        $userAgent = "AWSLambdaPSCore lib/AWSLambdaPSCore#$moduleVersion"
```

Comment:
In the normal case where we are appending the extra `AWSLambdaPSCore is redundant. How about this?

```
$userAgent = "lib/AWSLambdaPSCore#$moduleVersion"
if ($env:AWS_EXECUTION_ENV) {
	$env:AWS_EXECUTION_ENV += " $userAgent"
} else {
	$env:AWS_EXECUTION_ENV = "AWSLambdaPSCore " + $userAgent
}
```

Reply:
Done